util/config/config.go

```diff
@@ -117,13 +117,19 @@ func Init() error {
 			}
 			log.Printf("Mapped external port %d to internal port %d", externalPort, internalPort)
 
-			externalPort, internalPort, err = nat.AddPortMapping("tcp", int(Parameters.HttpWsPort), int(Parameters.HttpWsPort), "nkn", 10*time.Second)
+			externalPort, internalPort, err = nat.AddPortMapping(transport.GetNetwork(), int(Parameters.HttpWsPort), int(Parameters.HttpWsPort), "nkn", 10*time.Second)
```

Comment:
Which enum of transport.GetNetwork()?  ["tcp", "kcp", ?? ]

Reply:
Only tcp or udp